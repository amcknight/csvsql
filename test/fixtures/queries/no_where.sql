SELECT age, first_name
FROM test/fixtures/tables/people
