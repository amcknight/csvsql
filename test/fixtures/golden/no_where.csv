35, Andrew
36, Tom
31, Cayley
0, July
